lib/fluent/plugin/out_detect_exceptions.rb

```diff
@@ -42,6 +42,10 @@ class DetectExceptionsOutput < Output
     def configure(conf)
       super
 
+      if @remove_tag_prefix.empty?
+        abort("'remove_tag_prefix' is a required config option.")
```

Comment:
You don't want to exit Ruby if the parameter is not specified.. Just remove the `default` from the appropriate `config_param`, and fluentd will check for you (https://docs.fluentd.org/plugin-development/api-config-types). I'd also reorder the parameters to put the required one first.